src/Illuminate/Http/Request.php

```diff
@@ -255,6 +255,24 @@ public function has($key)
 	}
 
 	/**
+	 * Determine if the request contains a non-empty value for an input item matching a given pattern.
+	 *
+	 * @param  string  $pattern
+	 * @return bool
+	 */
+	public function hasRegex($pattern)
+	{
+		$input = $this->all();
+
+		foreach ($input as $key => $value)
```

Comment:
Welcome! :)

Can be. I just tried to follow the coding style of the class. For example:

https://github.com/KennedyTedesco/framework/blob/input-l5/src/Illuminate/Http/Request.php#L229